src/fm/usage.cljc

```diff
@@ -318,6 +344,32 @@
  (echo2)
  (exclaim2))
 
+(defm echo3
+  ^{:fm/args ::http-req
+    :fm/ret  ::http-resp
+    :fm/rel  (fn [{[req] :args resp :ret}] ; same interface as `:fn` in `s/fdef`
```

Comment:
@localshred I'm actually blown away that this is working already. All of a few weeks ago, `:fm/rel`-esque functionality was a distant hope at best. If you hadn't mapped all that `stest` territory, this probably wouldn't have come about at all, and certainly not so easily as it did. So epic :robot:.

Reply:
Pure energy. 💖